src/redux/reducers/childReducer.js

```diff
@@ -0,0 +1,34 @@
+import {
+  GET_CHILDREN_ACTION,
+  GET_CHILDREN_SUCCESS,
+  GET_COURSES_SUCCESS,
+  GET_COURSE_ACTION,
+} from '../actions/childAction';
+
+const initialState = {
+  child: {
```

Comment:
This should probably be an array that will be filled with child objects. I'd check with the endpoint (or wait for the updated endpoint to be merged and check that, but they will probably be very similar).